src/edu/ucsb/cs56/projects/games/roguelike/RogueController.java

```diff
@@ -363,26 +533,64 @@ public void keyPressed(KeyEvent key){
 	origX = x; 
 	origY = y;
 	switch (key.getKeyChar()){
-	case 'w'	    :	 this.y = this.y - logicHandler.getPlayer().getSpeed(); break;
-	case 'a'	    :  	 this.x = this.x - logicHandler.getPlayer().getSpeed(); break;
-	case 'd'	    :	 this.x = this.x + logicHandler.getPlayer().getSpeed(); break;
-	case 's'		:	 this.y = this.y + logicHandler.getPlayer().getSpeed(); break;
+	case 'w'	    :
+		this.y = this.y - logicHandler.getPlayer().getSpeed(); break;
+	case 'q'	    :
+		this.y = this.y - logicHandler.getPlayer().getSpeed();
+		this.x = this.x - logicHandler.getPlayer().getSpeed(); break;
```

Comment:
Now that these are multi-line, might as well throw the `break` on its own line.  It's easy to miss it this way.